src/python/WMCore/WMSpec/StdSpecs/Resubmission.py

```diff
@@ -83,6 +84,14 @@ def validateSchema(self, schema):
         Since we skip the master validation for Resubmission specs, we better have
         some specific validation
         """
+        #TODO: for the legacy code if ACDC is created before the updated. remove in next release (Mar 2018)
+        if schema.get('OriginalRequestType') == 'Resubmission':
+            # we cannot validate such schema
+            return
+
+        if schema.get("ResubmissionCount", 1) > 1:
```

Comment:
I'm not sure we need these lines. This logic follows exactly the same as OriginalRequestType, just in a different way :) The bottom line is, we cannot validate ACDC of ACDC (thus, OriginalRequestType=Resubmission, or even ResubmissionCount > 1)

Reply:
First line (OriginalRequestType=Resubmission) verifies ACDC of ACDC if the ACDC is created before this patch update.
second line (ResubmissionCount > 1) verifies if the ACDC is created after this patch update.

Both cases (ACDC of ACDC) we won't validate the parameters.